mappings/net/minecraft/text/LocalizedComponent.mapping

```diff
@@ -1,9 +1,9 @@
-CLASS jw net/minecraft/text/TranslatableTextComponent
+CLASS jw net/minecraft/text/LocalizedComponent
```

Comment:
Big :-1:, the real name for this is `TranslatableComponent` as seen by the `toString`.

Reply:
Hm I see...